pyproject.toml

```
@@ -0,0 +1,61 @@
+[tool.poetry]
+name = "geocoding-service"
+version = "0.1.0"
+description = "Geocoding as a service"
+authors = ["Togglecorp Dev"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+geopandas = "^1.0.1"
+fiona = "^1.10.1"
+fastapi = "^0.115.8"
+requests = "^2.32.3"
+uvicorn = "^0.34.0"
+geojson = ">=2.5.0"
+
+
+[tool.poetry.dev-dependencies]
```

```suggestion
[tool.poetry.group.dev.dependencies]
```

https://python-poetry.org/blog/announcing-poetry-2.0.0#poetry-core-200httpsgithubcompython-poetrypoetry-corereleasestag200